xc/common/utils/prjxray_arch_import.py

```diff
@@ -480,6 +480,18 @@ def get_tiles(
         SELECT pkey, grid_x, grid_y, phy_tile_pkey, tile_type_pkey, site_as_tile_pkey FROM tile
         """):
 
+        if phy_tile_pkey is not None:
+            c2.execute(
+                "SELECT prohibited FROM site_instance WHERE phy_tile_pkey = ?",
+                (phy_tile_pkey, )
+            )
+
+            prohibited_sites = c2.fetchall()
+
+            # Skip generation of prohibited tiles
+            if True in prohibited_sites:
```

Comment:
This won't work as expected.  fetchall returns [(True), (True)] not [True, True].

Reply:
What I've done is unpack the tuple at fetchall:
```
any_prohibited_sites = any(prohibited for (prohibited,) in c2.fetchall())
```

Reply:
Yep, much cleaner, thanks for the suggestion.

Reply:
Fixed